packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -40,6 +39,7 @@ import { emptyHeader } from 'src/navigator/Headers.v2'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import { StackParamList } from 'src/navigator/types'
+import { completePaymentRequest, declinePaymentRequest } from 'src/paymentRequest/actions'
```

Comment:
I realize this is ported code but I dont think `completePaymentRequest` is needed here

Reply:
Good catch, will remove!